tests/helpers/graphical_units.py

```diff
@@ -35,9 +36,9 @@ def set_test_scene(scene_object, module_name):
     config["frame_rate"] = 15
 
     with tempfile.TemporaryDirectory() as tmpdir:
-        os.makedirs(os.path.join(tmpdir, "tex"))
-        file_writer_config["text_dir"] = os.path.join(tmpdir, "text")
-        file_writer_config["tex_dir"] = os.path.join(tmpdir, "tex")
+        os.makedirs(Path(tmpdir) / "tex")
+        file_writer_config["text_dir"] = Path(tmpdir) / "text"
+        file_writer_config["tex_dir"] = Path(tmpdir) / "tex"
```

Comment:
In the past we have had issues with storing Path objects in file_writer_config. For now, I believe it's safer to use Path objects whenever possible, but store them as strings in the config dicts, until the config system is more stable.

Reply:
ok, done.